components/filter/filter.js

```diff
@@ -735,6 +735,12 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 	_handleClear() {
 		const dimension = this._getActiveDimension();
 
+		if (dimension.searchType !== 'none') {
+			this._handleClearSearch(dimension);
+			const searchInput = this.shadowRoot.querySelector('d2l-input-search');
+			if (searchInput) searchInput.value = '';
```

Comment:
This isn't great. I haven't been able so far to force [this search](https://github.com/BrightspaceUI/core/blob/main/components/filter/filter.js#L447) to re-render since its `value` technically isn't changing. Let me know any suggestions here.

Reply:
Is that because `dimension.searchValue` doesn't change until they execute the search? Could its value be set to empty?

Reply:
Yup I believe that's correct. It's getting set to empty string in `_handleClearSearch` but it already was empty so to filter nothing has changed.